### DISCUSSION TITLE
Spell checking

### DISCUSSION THREAD

**Discussion by ghost** [community/community#6479] [Other Feature Feedback, Questions, & Ideas]
Simply put - it'd be nice to have a simple spellcheck dictionary for the  github website wether it be when commenting or simply adding a title to a new issue.
far too many times have i mispelled the title by accident and not realised it ;P.

**Comment by orionnoir** [comment]
It is very important to quality prompting. Did this get addressed? I am using Microsoft Visual Studio Professional 2022 (2) (64-bit) - Current Version 17.6.3 and the Ask GitHub Copilot input fields inline in the IDE or in the separate tool view window did not have any helpful context tools (which should also be another feature, a way to hook prompt snippets into the system. The suggestions are helpful, but I can't just pull a part in, or make changes to the suggestion. It is a source of typo/cut & paste errors.
